openfeature/provider.go

```diff
@@ -66,6 +66,12 @@ type StateHandler interface {
 	Status() State
 }
 
+// TrackingHandler is the contract for tracking
+// FeatureProvider can opt in for this behavior by implementing the interface
+type TrackingHandler interface {
```

Comment:
I'm certainly OK with `TrackingHandler`, but I wonder why not just `Tracking` or `TrackingProvider`.

Interested in others' opinions on this @aepfli @thomaspoignant

Reply:
I think `Tracker` would be the go-idiomatic naming.

Reply:
I agree with `Tracker`, go likes simple names.